packages/dependencies/test/index.test.ts

```diff
@@ -15,13 +15,18 @@ limitations under the License.
 */
 
 import { expect, it } from "@solarwinds-apm/test"
+import { readFile } from "fs/promises"
 
-import { devDependencies } from "../package.json"
 import { dependencies } from "../src/index.js"
 
+const { devDependencies } = JSON.parse(
+  await readFile("package.json", { encoding: "utf-8" }),
+) as { devDependencies: Record<string, string> }
```

Comment:
Importing from `package.json` caused some issues with ESM on recent Node versions in tests, this works the same way.